File: Python/program/Shirt.py
# Import Class Induk
from Clothing import Clothing

# Deklarasi Class

class Shirt(Clothing):

    # Privat (__), Protected (_), Public Atributes
    __color = ""
    __sleevetype = ""

    # Konstruktor
    def __init__(self, color = "", sleevetype = ""):
        self.__color = color
        self.__sleevetype = sleevetype

    # Getter dan Setter
        
    # Get color
    def get_color(self):
        return self.__color
    
    # Set color
    def set_color(self, color):
        self.__color = color

    # Get sleevetype
    def get_sleevetype(self):
        return self.__sleevetype
    
    # Set sleevetype
    def set_sleevetype(self, sleevetype):
        self.__sleevetype = sleevetype

File: PHP/program/Shirt.php

<?php

// Memanggil kembali suatu file untuk mengambil suatu classnya
require ('Clothing.php');

// Deklarasi Class
class Shirt extends Clothing{
    private $color;
    private $sleevetype;

    // Konstruktor
    public function __construct($id, $name, $brand, $price, $size, $material, $gender, $color, $sleevetype) {
        parent::__construct($id, $name, $brand, $price, $size, $material, $gender); // Memanggil konstruktor dari parentnya
        $this->sleevetype = $sleevetype;
        $this->color = $color;
    }

    // Getter dan Setter
    
    // Get sleevetype
    public function get_sleevetype() {
        return $this->sleevetype;
    }

    // Set sleevetype
    public function set_sleevetype($sleevetype) {
        $this->sleevetype = $sleevetype;
    }

    // Get color
    public function get_color() {
        return $this->color;
    }

    // Set color
    public function set_color($color) {
        $this->color = $color;
    }
}